app/src/main/res/values/strings.xml

```diff
@@ -166,4 +168,5 @@
     <string name="re_authenticate_fail">"Failed to re-authenticate!"</string>
     <string name="update_username_success">"Successfully updated username</string>
     <string name="update_username_fail">Failed to update username</string>
+    <string name="starting_workout_while_another_is_active">Cannot start a workout while one is active</string>
```

Comment:
The name here should indicate that it's used for a toast. Also this is a very long message for a toast. Consider using another view to show it like an AlertDialog or a Snackbar.

Reply:
Decided to keep the toast because Vladi made a cool toastManager and just shortened the message👍

Reply:
But toasts aren't cool 😅  More and more apps are moving away from them. Perhaps the manager can be extended with other views like Snackbars?

Reply:
Ok will open another pr to extend the ToastManager to use Snackbars